internal/providers/aws.go

```diff
@@ -0,0 +1,171 @@
+// Copyright 2022 The OPA Authors.  All rights reserved.
+// Use of this source code is governed by an Apache2
+// license that can be found in the LICENSE file.
+
+package providers
+
+import (
+	"crypto/hmac"
+	"crypto/sha256"
+	"fmt"
+	"net/url"
+	"sort"
+	"strings"
+	"time"
+
+	"github.com/open-policy-agent/opa/ast"
+)
+
+func stringFromTerm(t *ast.Term) string {
+	if v, ok := t.Value.(ast.String); ok {
+		return string(v)
+	}
+	return ""
```

Comment:
[golf] 
```suggestion
	v, _ := t.Value.(ast.String)
	return string(v)
```
probably does the same thing, but it's less clear. let's keep as-is.